public/app/plugins/datasource/prometheus/partials/config.html

```diff
@@ -48,3 +48,23 @@
   </div>
 </div>
 
+<h3 class="page-heading">Misc</h3>
+<div class="gf-form-group">
+  <div class="gf-form-inline">
+    <div class="gf-form max-width-30">
```

Comment:
I'd make this wider.

Reply:
max widths css are defined in `public/sass/utils/_widths.css` from 1 through 30. Increase this will generate a lot of useless css definitions..
I've tried with a bigger width (up to 50) and the text field was not aligned with the URL field. I prefer when it's aligned :)